### PR TITLE
[DX-1206] Avoid potential confusion with next LTS version before announcement

### PR DESCRIPTION
**Pull request by andyo-tyk**
### Preview Link
[LTS preview link](https://deploy-preview-4312--tyk-docs.netlify.app/docs/nightly/developer-support/long-term-support-releases)
### Description
Removed unconfirmed LTS version numbers to avoid potential confusion

___

## **Type**
documentation

___

## **Description**
- Updated the Long-Term Support (LTS) version information to remove specific mention of version 5.3 LTS, replacing it with a placeholder for the next LTS version (LTS+1, version TBC).
- Added a note indicating that the next LTS version will be announced in April 2024.
- Temporarily commented out the compatibility matrix for the unannounced LTS version to avoid potential confusion before the official announcement.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>long-term-support-releases.md</strong><dd><code>Update LTS Version Information and Compatibility Matrix</code>&nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/developer-support/long-term-support-releases.md
<li>Removed specific mention of version 5.3 LTS and replaced it with a <br>placeholder for the next LTS version.<br> <li> Added a note that the next LTS version will be announced in April <br>2024.<br> <li> Commented out the compatibility matrix for the unannounced LTS <br>version.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4312/files#diff-27a983dee8887590907b359f2077b64f02222c1605a0ac5dab20f7eaeadeb6f4">+9/-4</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/developer-support/long-term-support-releases.md [hunk 1]
```diff
@@ -105,19 +105,24 @@ As part of the release for the new Gateway LTS version we will commit to show ev
 1. **Recommended releases** - To ensure you get the most out of the latest Tyk experience, we'll provide information on which versions of different components across the entire stack you need.
 2. **Backwards Compatibility** - We'll provide information on what components and versions remain backwards compatible with the new Tyk Dashboard and Tyk Gateway versions.
 
-Our next LTS version is 5.3. The table below shows recommended compatibility:
+Our next LTS version will be announced in April 2024.
+
+{{/* COMMENTED OUT UNTIL LTS ANNOUNCED
```
Comment (andyo-tyk) [inline tyk-docs/content/developer-support/long-term-support-releases.md]:
```suggestion
<!-- COMMENTED OUT UNTIL LTS ANNOUNCED
```

File: tyk-docs/content/developer-support/long-term-support-releases.md [hunk 2]
```diff
@@ -105,19 +105,24 @@ As part of the release for the new Gateway LTS version we will commit to show ev
...
 |         | Operator v0.17 | Operator v0.16 |
 |         | Sync v1.4.3   | Sync v1.4.3 |
 |         | Helm Chart (tyk-stack, tyk-oss, tyk-dashboard, tyk-gateway) v1.3.0 | Helm all versions |
 | | EDP v1.8.3 | EDP all versions |
 | | Pump v1.9.0 | Pump all versions |
 | | TIB (if using standalone) v1.5.1 | TIB all versions |
 
-The compatibility matrix table shown above will be part of upcoming [Gateway release notes]({{< ref "product-stack/tyk-gateway/release-notes/overview" >}}) for versions 5.3 and beyond. Additionally, these release notes will list tested third-party dependencies like *PostgreSQL, MongoDB, Redis* and more.
+The compatibility matrix table shown above will be part of upcoming [Gateway release notes]({{< ref "product-stack/tyk-gateway/release-notes/overview" >}}) for versions x.x and beyond. Additionally, these release notes will list tested third-party dependencies like *PostgreSQL, MongoDB, Redis* and more.
+*/}}
```
Comment (andyo-tyk) [inline tyk-docs/content/developer-support/long-term-support-releases.md]:
```suggestion
-->
```

**Review by dcs3spp — APPROVED** [review]
PR LGTM! @allaneyles Could you check the preview link given in the description of the PR and if all ok, then I will merge and release

**Review by allaneyles — APPROVED** [review]

**Pull request merged**